packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import { AccountStatusResponse, GetAccountStatus } from '@deriv/api-types';
+import { Button, Loading } from '@deriv/components';
+import { WS, getPlatformRedirect, platforms } from '@deriv/shared';
+import { TCoreStores } from '@deriv/stores/types';
+import { Localize } from '@deriv/translations';
+import Expired from '../../../Components/poa/status/expired';
+import NeedsReview from '../../../Components/poa/status/needs-review';
+import NotRequired from '../../../Components/poa/status/not-required';
+import PoaStatusCodes from '../../../Components/poa/status/status-codes';
+import ProofOfAddressForm from './proof-of-address-form';
+import Submitted from '../../../Components/poa/status/submitted';
+import Unverified from '../../../Components/poa/status/unverified';
+import Verified from '../../../Components/poa/status/verified';
+import { populateVerificationStatus } from '../Helpers/verification.js';
+
+type TProofOfAddressContainer = {
+    is_mx_mlt?: boolean;
+    is_switching?: boolean;
+    has_restricted_mt5_account?: boolean;
+    refreshNotifications: TCoreStores['notifications']['refreshNotifications'];
+    app_routing_history: TCoreStores['portfolio']['error']['app_routing_history'];
+};
+
+type TAuthenticationStatus = Record<
+    | 'allow_document_upload'
+    | 'allow_poi_resubmission'
+    | 'allow_poa_resubmission'
+    | 'is_age_verified'
+    | 'has_poi'
+    | 'has_submitted_poa'
+    | 'needs_poa'
+    | 'needs_poi'
+    | 'poa_address_mismatch'
+    | 'resubmit_poa',
+    boolean
+> & { document_status?: DeepRequired<GetAccountStatus>['authentication']['document']['status'] };
+
+const ProofOfAddressContainer = ({
+    is_mx_mlt,
+    is_switching,
+    has_restricted_mt5_account,
+    refreshNotifications,
+    app_routing_history,
```

Comment:
can't we extract these props from store directly using useStore instead of passing as props?